### DISCUSSION TITLE
"status pending" in console

### DISCUSSION THREAD

**Discussion by proxymv** [rsuite/rsuite#3831] [General]
I have started getting "status pending" from rsuite in the browser console. 
Browser sources show that it is from the Avatar component. 
Not a major problem, just that it keeps printing that.

**Comment by simonguo** [comment]
Do you have a screenshot of the error?

**Reply by proxymv** [reply]
Not really an error I think. 

![Screenshot 2024-06-04 at 14 31 16](https://github.com/rsuite/rsuite/assets/109961989/efa476f0-f7fb-4db8-92f8-39fa35e46e25)
![Screenshot 2024-06-04 at 14 31 34](https://github.com/rsuite/rsuite/assets/109961989/47c44cea-7b59-49b6-b634-d038c6070cab)

**Reply by simonguo** [reply]
This debug info has been fixed in v5.59.1, you can upgrade to the latest version and try again.

**Reply by proxymv** [reply]
Yes. It is gone. Thank you very much.
